example/app/missing-data.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import { Area, Bar, CartesianChart, Line, Scatter } from "victory-native";
+import { useFont } from "@shopify/react-native-skia";
+import { appColors } from "./consts/colors";
+import inter from "../assets/inter-medium.ttf";
+import { Button } from "../components/Button";
+
+export default function MissingDataScreen() {
+  const font = useFont(inter, 12);
+  const [data, setData] = React.useState(DATA);
+
+  return (
+    <SafeAreaView style={styles.safeView}>
+      <View style={{ flex: 1, maxHeight: 400, padding: 32 }}>
+        <CartesianChart
```

Comment:
If we're going to have the option of "filling in" for skipped data, worth doing a toggle on this demo screen?

Reply:
That's a good idea, I'll add that in. I enabled this screen only in dev mode to just test things out – but I should add the toggle, would help with prototyping/testing this out.